.github/workflows/pathogen-repo-ci.yaml

```diff
@@ -238,6 +238,16 @@ jobs:
           ref: ${{ needs.workflow-context.outputs.sha }}
           path: ${{ env.NEXTSTRAIN_GITHUB_DIR }}
 
+      - name: Verify nextstrain-pathogen.yaml file
+        continue-on-error: false
```

Comment:
_nit_

This is the default, and we don't make it explicit elsewhere. I'd drop the line.